config/database.yml

```diff
@@ -0,0 +1,90 @@
+# PostgreSQL. Versions 9.3 and up are supported.
+#
+# Install the pg driver:
+#   gem install pg
+# On macOS with Homebrew:
+#   gem install pg -- --with-pg-config=/usr/local/bin/pg_config
+# On macOS with MacPorts:
+#   gem install pg -- --with-pg-config=/opt/local/lib/postgresql84/bin/pg_config
+# On Windows:
+#   gem install pg
+#       Choose the win32 build.
+#       Install PostgreSQL and put its /bin directory on your path.
+#
+# Configure Using Gemfile
+# gem "pg"
+#
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  username: thecodechaser
+  password: thecode123
+  host: localhost
```

Comment:
Should not be dependent on username or password:
If this is the case maybe remove that in commits and put in Doc to add username and password or now have db creation dependent on this. Should be able to run rails db:create without having to specify username and password in this config file directly. 

Bad practice to have username and password committed to a repo.